### ISSUE TITLE
#-431009714------------- Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1857]
<strong>PageName:</strong> FeedViewDetailsPage  

<strong>Source:</strong>`<calcite-button class="toggle-button" aria-label="App Launcher" alignment="center" appearance="transparent" kind="neutral" scale="m" type="button" width="auto" calcite-hydrated="">`

<strong>Element Location: </strong>arcgis-app-switcher,.toggle-button

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  aria-label attribute cannot be used on a calcite-button with no valid role attribute.

<strong>Tags:</strong> cat.aria,wcag2a,wcag412

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/aria-allowed-attr?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedViewDetailsPage\-431009714_Elements must only use allowed ARIA attributes.png`
